packages/plate-utils/src/useFormInputProps.ts

```diff
@@ -0,0 +1,68 @@
+interface InputProps {
+  /**
+   * Should we activate the onKeyDownCapture handler to preventDefault when
+   * the user presses enter?
+   */
+  preventDefaultOnEnterKeydown?: boolean;
+  /**
+   * A user provided callback which will be called after event.preventDefault
+   *
+   * @param e The original event
+   * @returns nothing
+   */
+  onKeyDownCaptureCallback?: (e: React.KeyboardEvent<HTMLDivElement>) => void;
```

Comment:
not needed, this can be done client side

Reply:
Not sure I fully understand this amend, sorry. I added that so that the user could use the pre-defined callback to `preventDefault` and add in their own callback as well. Is your suggestion to either let the user use the predefined `preventDefault` OR use their own callback?

Reply:
My comment was just on `onKeyDownCaptureCallback`. `preventDefaultOnEnterKeydown` is a good fit. Since the component is owned by the consumer, he can provide his own values inside them directly, instead of adding props.